samples/oauth_rails/VetsApiClaims/app/views/layouts/application.html.erb

```diff
@@ -68,6 +68,9 @@
                   <li class="site-header__nav-item">
                     <%= link_to 'Benefits Intake', documents_path, class: 'site-header__nav-item__link' %>
                   </li>
+                  <li class="site-header__nav-item">
+                    <%= link_to 'Power Of Attorney', documents_path, class: 'site-header__nav-item__link' %>
```

Comment:
if this is using the same path as Benefits intake, should it be one link with a better name?

Reply:
it wasn't this was a typo that's been fixed